client/header/activity-panel/style.scss

```diff
@@ -186,12 +185,10 @@
 .woocommerce-layout__activity-panel-mobile-toggle {
 	margin-right: 10px;
 	max-width: 48px;
-	height: $header-height;
-	position: absolute;
```

Comment:
Nice!  Love the removal of this.  It really cleans up the styling issues in the header.